### PR TITLE
Bump qt min version and remove code slated for removal in 0.13.0

### PR DESCRIPTION
**Pull request by j9ac9k**
In preparation for the 0.13.0 release, we are bumping up the minimum Qt required version to 5.15/6.2+ (from 5.12+/6.1+).  

### TODO

- [x] Change Qt version check to a warning from a raised exception
- [x] Undo removal of code that was needed for Qt 5.12 support (can talk about removing after the next release)
- [ ] ~Consider emitting warnings on if-checks for newer Qt versions.~
- [x] Remove lazyGetattr in `Qt/__init__.py` as we no longer support accessing QtWidgets via QtGui

### Files that need updates
    
Confirm the following files have been either updated or there has been a determination that no update is needed.

- [x] `README.md`
- [ ] `setup.py` N/A
- [x] `tox.ini`
- [x] `.github/workflows/main.yml` and associated `requirements.txt` and conda `environemt.yml` files
- [ ] `pyproject.toml` N/A
- [ ] `binder/requirements.txt` N/A

### CODE REVIEW

File: pyqtgraph/opengl/GLViewWidget.py [hunk 1]
```diff
@@ -381,13 +381,6 @@ def pan(self, dx, dy, dz, relative='global'):
...
         False (global). These values are deprecated but still recognized.
         """
-        # for backward compatibility:
-        if isinstance(relative, bool):
-            warnings.warn(
-                "'relative' as a boolean is deprecated, and will not be recognized in 0.13. "
-                "Acceptable values are 'global', 'view', or 'view-upright'",
-                DeprecationWarning, stacklevel=2
-            )    
         relative = {True: "view-upright", False: "global"}.get(relative, relative)
```
Comment (pijyoi) [inline pyqtgraph/opengl/GLViewWidget.py:384]: This line that converts boolean to its equivalent string argument should be removed too

Comment (j9ac9k) [reply pyqtgraph/opengl/GLViewWidget.py:384]: nice catch

File: pyqtgraph/opengl/GLViewWidget.py [hunk 2]
```diff
@@ -381,14 +381,6 @@ def pan(self, dx, dy, dz, relative='global'):
         Prior to version 0.11, *relative* was expected to be either True (x-aligned) or
         False (global). These values are deprecated but still recognized.
```
Comment (pijyoi) [inline pyqtgraph/opengl/GLViewWidget.py:382]: This documentation also needs to be removed

Comment (j9ac9k) [reply pyqtgraph/opengl/GLViewWidget.py]: Nice catch, removed!

**Comment by j9ac9k** [comment]
Now need to remove those pesky unused `warning` module imports

**Pull request merged**